demoapp/run_demo_app.py

```diff
@@ -0,0 +1,116 @@
+"""
+Implements bill summarization, and category and tags generation for all bills (no MGL sections data is available for ~1300 bills)
+All the code implemented here is located in demoapp.py
+This code runs very slow as it requires accessing data from a large file
+"""
+
+import streamlit as st
+import pandas as pd
+from sklearn.feature_extraction.text import TfidfVectorizer
+from sklearn.metrics.pairwise import cosine_similarity
+from rouge_score import rouge_scorer
+from sentence_transformers import CrossEncoder
+from sidebar import *
```

Comment:
Not a big deal, but best to avoid `import *` to avoid polluting or accidentally overlapping the namespace.